### DISCUSSION TITLE
How do I recover my PASSWORD VIDEO DOWNLOADER HELPER CHANGE LAPTOP

### DISCUSSION THREAD

**Discussion by Francosist** [aclap-dev/video-downloadhelper#980] [Help and Q&A]
### Your question

How do I recover my PASSWORD VIDEO DOWNLOADER HELPER CHANGE LAPTOP

### Addon & CoApp version and details

How do I recover my PASSWORD VIDEO DOWNLOADER HELPER CHANGE LAPTOP

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

In order to get proper help, you need to realize that VDH does not have a password.  It has a license.  So when you are searching for help, "license" is the word you need to use.

Whenever you have a question, it should be a routine thing for you to click the Wiki link that appears at the top of every page of this forum.  So click that Wiki link.  That will take you to the closest thing we have to a user manual.  There is a table of contents down the right side of the page.  Click the chapter for FAQ.  You will find the answer to your question there.

**Comment by jcfrog** [comment]
you can use https://www.downloadhelper.net/recover-license or send us an email at licenses@downloadhelper.net
